akka-projection-persistence-query/src/main/scala/akka/persistence/query/TypedEventEnvelope.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.query
```

Comment:
Keeping it on same package as original envelope. Eventually we can move it into Akka and have a typed adapter for the query traits (`EventsByTagQuery`, `CurrentEventsByTagQuery`) that return a `Source[TypedEventEnvelope, _]`.

Reply:
Using the same package name as another module will cause problems to Java's module system, I think (OSGi definitely doesn't allow it).

Reply:
Any suggestion for package name? 

- [ ] `akka.projection.journal` ?
- [ ] `akka.projection.persistence.query` ?
- [ ] add yours here..

Reply:
if we go with calling it `EventSourcedProvider` we can use `akka.projection.eventsourced`.

`akka-persistence-query` almost becomes an implementation detail that the user doesn't have to care about.